src/containers/Layout/Sidebar.js

```diff
@@ -11,6 +11,7 @@ import ConnectButton from 'components/Basic/ConnectButton';
 import { Label } from 'components/Basic/Label';
 import { connectAccount, accountActionCreators } from 'core';
 import logoImg from 'assets/img/logo.png';
+import prdtImg from 'assets/img/prdt.png';
```

Comment:
The logo is huge, maybe scale it down a bit? 111KiB seems not terrible but still quite a lot for such a small thingie...